python/py-rchitect/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        randy3k rchitect 0.3.40 v
+name                py-rchitect
+revision            0
+license             MIT
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Mapping R API to Python
+long_description    {*}${description}
+
+checksums           rmd160  2e59a9a673c7edf6a1b61b12ce074b1dd474df4f \
+                    sha256  c1edf37c59ed66c044724b7f1a9d4542f3f9d2e05109a6413b10cce64c584b05 \
+                    size    36456
+
+depends_build-append \
+                    port:py${python.version}-pytest-runner \
+                    port:py${python.version}-setuptools
+depends_lib-append  port:py${python.version}-cffi \
+                    port:py${python.version}-six
+
+# Fails with python311: python -m installer: error: the following arguments are required: wheel
+# https://github.com/randy3k/rchitect/issues/23
```

Comment:
this comment isn't accurate as it does build fine with Python 3.11. Please remove and then we should be good to merge.

Reply:
You are absolutely right, I just forgot to remove it.